README.md

```diff
@@ -7,19 +7,21 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client in the style of the classic one.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby.
 
-### Features
+## Features
 
-* Classic F-Droid style, without cards or inappropriate animations
-* Fast repositories synchronization under good network conditions
-* Built upon standard Android components and a minimal set of dependencies
-* Made with attention to details, both in program logic and design
+* Classic F-Droid style.
+* No cards or inappropriate animations.
+* All apps shown in "Available".
+* Fast repository syncing.
+* Standard Android components and minimal dependencies.
+* Attention to detail in programming logic and design.
```

Comment:
"detail**s**"

Reply:
Actually, I may be not right, seems "attention to detail" is a correct way too. But since I mention "program login _and_ design", detail**s** may be a better option. Or not.

Reply:
You could be, but in my head that is "Attention to the details of ___."
"Attention to detail" is a figure of speech, so I think it is correct.
"Pedantry in" or "Pedantic __" should be a better option, but isn't.

Reply:
>  "Attention to detail" is a figure of speech, so I think it is correct.

Yes, I know (tbh, didn't know about it when opened this review, but then searched for it). I wanted to suggest another option since for me it looks more clean (again, it's my sense of English: if I translated this line to Russian, "detail" will be translated to "details").

Closing this.